publication-commons/src/test/resources/features/file/file_access_read.feature

```diff
@@ -68,28 +75,84 @@ Feature: File metadata read and file download permissions
+      | InternalFile        | Editor                      | Allowed     |
 
       | HiddenFile          | Authenticated               | Not Allowed |
       | HiddenFile          | Contributor                 | Not Allowed |
       | HiddenFile          | Publishing curator          | Not Allowed |
       | HiddenFile          | Not related external client | Not Allowed |
+      | HiddenFile          | Editor                      | Not Allowed |
+
+  Scenario Outline: Verify file metadata read permissions when user is not from file owners institution or contributing institution
+    Given a file of type "<FileType>"
+    And the file is owned by "publication creator"
+    When the user have the role "<UserRole>"
+    And the user belongs to "non curating institution"
+    And the user attempts to "read-metadata"
+    Then the action outcome is "<Outcome>"
+
+    Examples:
+      | FileType            | UserRole           | Outcome     |
+      | UploadedFile        | Authenticated      | Not Allowed |
+      | UploadedFile        | Contributor        | Not Allowed |
+      | UploadedFile        | Publishing curator | Not Allowed |
+      | UploadedFile        | Editor             | Not Allowed |
+
+      | PendingOpenFile     | Authenticated      | Not Allowed |
+      | PendingOpenFile     | Contributor        | Allowed     |
+      | PendingOpenFile     | Publishing curator | Not Allowed |
+      | PendingOpenFile     | Editor             | Allowed     |
+
+      | PendingInternalFile | Authenticated      | Not Allowed |
+      | PendingInternalFile | Contributor        | Allowed     |
+      | PendingInternalFile | Publishing curator | Not Allowed |
+      | PendingInternalFile | Editor             | Not Allowed |
+
+      | OpenFile            | Authenticated      | Allowed     |
+      | OpenFile            | Contributor        | Allowed     |
+      | OpenFile            | Publishing curator | Allowed     |
+      | OpenFile            | Editor             | Allowed     |
+
+      | InternalFile        | Authenticated      | Not Allowed |
+      | InternalFile        | Contributor        | Allowed     |
+      | InternalFile        | Publishing curator | Not Allowed |
+      | InternalFile        | Editor             | Allowed     |
+
+      | HiddenFile          | Authenticated      | Not Allowed |
+      | HiddenFile          | Contributor        | Not Allowed |
+      | HiddenFile          | Publishing curator | Not Allowed |
+      | HiddenFile          | Editor             | Not Allowed |
+
+
+  Scenario: Verify file download permissions when user is not contributing institution and PendingOpenFile has embargo
+    Given a file of type "PendingOpenFile"
+    And the file is owned by "publication creator"
+    And the file has embargo
+    When the user have the role "Editor"
+    And the user belongs to "non curating institution"
+    And the user attempts to "download"
+    Then the action outcome is "Not Allowed"
```

Comment:
Do we have similar test when editor belongs to publication and file has embargo?